### DISCUSSION TITLE
Looking for Ant Media Haproxy configuration file

### DISCUSSION THREAD

**Discussion by ugacomp** [ant-media/Ant-Media-Server#5981] [Q&A]
Hello
I am looking for the Ant Media Haproxy configuration file similar to the Nginx load balancing configuration you provided. 
Would you mind providing a full setup file, that's easier to navigate?

**Comment by USAMAWIZARD** [comment]
here is the Config file from HA Proxy.
https://antmedia.io/docs/guides/clustering-and-scaling/load-balancing/load-balancer-with-haproxy-ssl-termination/#configuring-haproxy

Please let me know if this is what you are looking for or something else.
